roles/keystone-setup/tasks/main.yml

```diff
@@ -55,7 +55,8 @@
       password: "{{ secrets.admin_password }}"
   with_items: "{{ keystone.users }}"
   # If cloud_admin already exists, don't set the cloud_admin user/pass (overwriting existing userpass)
-  when: "item.name != 'cloud_admin' or (item.name == 'cloud_admin' and 'cloud_admin' not in user_facts.ansible_facts.openstack_users|map(attribute='username')|list)"
+  when: "item.name not in ['cloud_admin', 'monitor'] or
+         (item.name in ['cloud_admin', 'monitor'] and item.name not in user_facts.ansible_facts.openstack_users|map(attribute='username')|list)"
```

Comment:
Looking at this again.

What we want is to create the user if it doesn't exist, but not modify the user if it does exist.

The first part of the conditional, that may actually prevent the initial creation, if I'm reading that right.

Paul, can you unroll this logical set in plain english to walk us through it?